blaze/compute/sql.py

```diff
@@ -111,6 +111,16 @@ def compute_up(t, s, **kwargs):
     return compute(t._scalar_expr, d, post_compute=False).label(name)
 
 
+@dispatch(Concat, (Select, Selectable), (Select, Selectable))
+def compute_up(t, lhs, rhs, **kwargs):
+    if t.axis != 0:
+        raise NotImplementedError(
+            'Cannot concat along a non-zero axis in sql',
```

Comment:
I wonder if pointing people to `Merge` expressions is useful in this case since if they pass `axis=1`, `Merge` is likely what they want.